projects/js-packages/components/components/spinner/stories/index.jsx

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable react/react-in-jsx-scope */
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Spinner from '../index.jsx';
+
+export default {
+	title: 'Playground/Spinner',
+	component: Spinner,
+};
+
+const Template = args => <Spinner { ...args } />;
+
+const DefaultArgs = {
+	moduleName: 'The Module Name',
```

Comment:
This seems to be a leftover from the Footer component, I think we should remove it.